Podfile.lock

```diff
@@ -1,16 +1,42 @@
 PODS:
-  - Pelias (1.0.0-beta1):
-    - Pelias/Core (= 1.0.0-beta1)
-  - Pelias/Core (1.0.0-beta1)
+  - OnTheRoad (0.1)
```

Comment:
Yeah. It's a recommended thing these days. http://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control